tests/foreman/api/test_docker.py

```diff
@@ -1083,146 +1083,77 @@ def setUpClass(cls):
             url=EXTERNAL_DOCKER_URL,
         ).create()
 
-    def setUp(self):
-        """Generate default ``DockerHubContainer`` keyword arguments.
-
-        Generate a dict of arguments that can be used when instantiating a
-        ``DockerHubContainer`` and save it as ``self.dhc_kwargs``.
-
-        """
-        self.container_name = gen_string('alphanumeric')
-        self.dhc_kwargs = {
```

Comment:
My imo is it's not worth to create separate setup section to save 1 line, especially if we're talking about dict initialization, not about some entity creation (which could actually save some time). If setup section will be actually needed in future - than we can easily add it in future, no need to introduce some redundancy right now :)
But that's just my imo, we can discuss it in IRC if you wish.